pallets/rmrk-core/src/tests.rs

```diff
@@ -70,33 +70,33 @@ fn mint_nft_works() {
 			Origin::signed(ALICE),
 			ALICE,
 			0,
-			ALICE,
-			Permill::from_float(20.525),
+			Some(ALICE),
```

Comment:
Currently there is conflict with `main` related to non optional. Recipient and royalty are non optional now so this change should be reverted.

Reply:
fixed with [ca15374a9814222fee117d44cda339ea31d84963](https://github.com/rmrk-team/rmrk-substrate/commit/ca15374a9814222fee117d44cda339ea31d84963#diff-53089f3950b3630a05b64ac0a7a0d3cc3d9b5d787a87f5e325f5c489bd05c37c)